src/deploy/osp_deployer/deployer.py

```diff
@@ -231,6 +235,16 @@ def deploy():
                                     settings.director_install_account_pwd,
                                     cmd)[0]
         logger.debug("=== Overcloud stack state : " + overcloud_status)
+        if settings.hpg_enable:
+            logger.info(
+                " HugePages has been successfully configured with size: " +
+                settings.hpg_size)
+        if settings.numa_enable:
+            logger.info(
+                " NUMA has been successfully configured"
+                " with hostos_cpus count: " +
+                settings.hostos_cpu_count)
+
```

Comment:
i beleive the the above is already in Directory.py :: summarize_deployment

Reply:
Removed